Week3/homework/1-step3.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
 
 function foo(func) {
-  // What to do here? 
-}
+  const x = 12;
```

Comment:
I'm guessing these two lines were testing something else out. 😉 
They don't really have a place in this exercise.